api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/core/OrderEntryConcepts.java

```diff
@@ -1,103 +0,0 @@
-        public final static String WHEN_REQUIRED = "160857AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
-        public final static String IMMEDIATELY = "3cdc5794-26fe-102b-80cb-0017a47871b2";
-
-        public final static String TABLET = "1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
-        public final static String CAPSULE = "1608AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
-    }
-
-    @Override
-    public int getVersion() {
-        return 1;
-    }
-
-    @Override
-    protected void installNewVersion() throws Exception {
-        installOrderFrequencies();
-
-        // concepts are currently included in the HUM Dispensing MDS package
-    }
-
-    private void installOrderFrequencies() {
-        install(orderFrequency(Concepts.ONCE_A_DAY, 1.0, "fb8068a0-7647-4aec-93d0-14658533dfdf"));
-        install(orderFrequency(Concepts.IN_THE_MORNING, 1.0, "682a53f3-5614-4e64-88bd-42855cff4be3"));
-        install(orderFrequency(Concepts.AT_NIGHT, 1.0, "19c83062-7e1f-4d3f-8b97-335d089666bd"));
-        install(orderFrequency(Concepts.TWICE_A_DAY, 2.0, "e08de81e-3bed-4471-a26f-180417d543d9"));
-        install(orderFrequency(Concepts.THREE_TIMES_A_DAY, 3.0, "e51a7a00-7f29-410b-b8ab-2c3cd6a2a5ad"));
-        install(orderFrequency(Concepts.FOUR_TIMES_A_DAY, 4.0, "89765cb8-4539-45e5-90bb-9ac8272f8972"));
-        install(orderFrequency(Concepts.FIVE_TIMES_A_DAY, 5.0, "f96d8e96-754c-4858-980e-b53d7ff3fec1"));
-        install(orderFrequency(Concepts.SIX_TIMES_A_DAY, 6.0, "bb3a1445-36c9-4fdb-a1bc-ca52a5c687a9"));
-        install(orderFrequency(Concepts.SEVEN_TIMES_A_DAY, 7.0, "3a3c6594-a0c7-42c7-9027-b7ec2a4404e0"));
-        install(orderFrequency(Concepts.EIGHT_TIMES_A_DAY, 8.0, "227e3c01-5343-448e-9611-4af1336b2e4e"));
-        install(orderFrequency(Concepts.NINE_TIMES_A_DAY, 9.0, "902b2159-422d-4b3c-b71f-bda6ea5b7814"));
-        install(orderFrequency(Concepts.EVERY_TWO_HOURS, 2d/24d, "63c4bafb-ae18-4ce6-9cd3-13c4b7a54e51"));
-        install(orderFrequency(Concepts.EVERY_THREE_HOURS, 0.125, "a0667be8-f9c0-4b5b-afcd-6bf29c6497fe"));
-        install(orderFrequency(Concepts.EVERY_FOUR_HOURS, 4d/24d, "34dbf28e-da5c-4c63-a7fd-451c9d33a59a"));
-        install(orderFrequency(Concepts.EVERY_SIX_HOURS, 0.25, "26921f5f-2188-494e-af52-9373c54adca7"));
-        install(orderFrequency(Concepts.EVERY_TWELVE_HOURS, 0.5, "0d99917c-ee7b-49e7-b55c-5d2457d9c40a"));
-        install(orderFrequency(Concepts.WHEN_REQUIRED, null, "da558e86-3fbc-427a-8e4e-06d61c38d813"));
-        install(orderFrequency(Concepts.IMMEDIATELY, null, "8487d0db-3378-49b8-b24c-18f121dfbfd0"));
-    }
-
-    /**
-     * OrderFrequency is not allowed to be resaved after it has an order pointing to it, or it will throw the exception:
-     * "org.openmrs.api.APIException: This order frequency cannot be edited because it is already in use"
-     *
-     * This method only calls the base install method if a field has actually changed, to prevent against spuriously
-     * getting this error.
```

Comment:
@mseaton see comment above... will Iniz be smart enough to not run into this?  (I guess you probably already tested)

Reply:
I don't have any idea actually

Reply:
We will have to keep an eye on this, then, I guess... if your dev environments didn't have any orders in them you may not have run into this?

Reply:
Good catch.  I created an issue in initailizer github, and a PR/branch that demonstrates it, and created an issue in our JIRA to keep track of this.